yarn-project/circuits.js/src/structs/__snapshots__/base_rollup.test.ts.snap

```diff
@@ -248,141 +311,82 @@ next_available_leaf_index: 512
 [ leaf_value: 0x1000
 next_value: 0x1100
-next_index: 1200
- leaf_value: 0x1001
-next_value: 0x1101
-next_index: 1201
- leaf_value: 0x1002
-next_value: 0x1102
-next_index: 1202
- leaf_value: 0x1003
-next_value: 0x1103
-next_index: 1203
- leaf_value: 0x1004
-next_value: 0x1104
-next_index: 1204
- leaf_value: 0x1005
-next_value: 0x1105
-next_index: 1205
- leaf_value: 0x1006
-next_value: 0x1106
-next_index: 1206
- leaf_value: 0x1007
-next_value: 0x1107
-next_index: 1207
+next_index: 0x120000000000000000000000000000000000000000000000000000000000
+ leaf_value: 0x100100000000000000000000000000000000000000000000000000000000
+next_value: 0x110100001201000000000000000000000000000000000000000000000000
+next_index: 0x1002000000000000000000000000000000000000000000000000
+ leaf_value: 0x1102000012020000000000000000000000000000000000000000
+next_value: 0x10030000000000000000000000000000000000000000
+next_index: 0x11030000120300000000000000000000000000000000
+ leaf_value: 0x100400000000000000000000000000000000
+next_value: 0x110400001204000000000000000000000000
+next_index: 0x1005000000000000000000000000
+ leaf_value: 0x1105000012050000000000000000
+next_value: 0x10060000000000000000
+next_index: 0x11060000120600000000
+ leaf_value: 0x100700000000
+next_value: 0x110700001207
+next_index: 0x200000000000000000000000000000000000000000000000000000000000
+ leaf_value: 0x200000000000000000000000000000000000000000000000000000000000
+next_value: 0x200100000000000000000000000000000000000000000000000000000000
+next_index: 0x200200000000000000000000000000000000000000000000000000000000
+ leaf_value: 0x200300000000000000000000000000000000000000000000000000000000
+next_value: 0x200400000000000000000000000000000000000000000000000000000000
+next_index: 0x200500000000000000000000000000000000000000000000000000000000
  ]
 low_nullifier_membership_witness:
-[ leaf_index: 2000
-sibling_path: [ 0x2000 0x2001 0x2002 0x2003 0x2004 0x2005 0x2006 0x2007 ]
- leaf_index: 2001
-sibling_path: [ 0x2001 0x2002 0x2003 0x2004 0x2005 0x2006 0x2007 0x2008 ]
- leaf_index: 2002
-sibling_path: [ 0x2002 0x2003 0x2004 0x2005 0x2006 0x2007 0x2008 0x2009 ]
- leaf_index: 2003
-sibling_path: [ 0x2003 0x2004 0x2005 0x2006 0x2007 0x2008 0x2009 0x200a ]
- leaf_index: 2004
-sibling_path: [ 0x2004 0x2005 0x2006 0x2007 0x2008 0x2009 0x200a 0x200b ]
- leaf_index: 2005
-sibling_path: [ 0x2005 0x2006 0x2007 0x2008 0x2009 0x200a 0x200b 0x200c ]
- leaf_index: 2006
-sibling_path: [ 0x2006 0x2007 0x2008 0x2009 0x200a 0x200b 0x200c 0x200d ]
+[ leaf_index: 2006
+sibling_path: [ 0x2007 0x200100000000000000000000000000000000000000000000000000000000 0x200100000000000000000000000000000000000000000000000000000000 0x200200000000000000000000000000000000000000000000000000000000 0x200300000000000000000000000000000000000000000000000000000000 0x200400000000000000000000000000000000000000000000000000000000 0x200500000000000000000000000000000000000000000000000000000000 0x200600000000000000000000000000000000000000000000000000000000 ]
```

Comment:
This sort of change in the snapshots is usually because something got misaligned in the serialization. `0x200100000000000000000000000000000000000000000000000000000000` is not a value we are generating in the factories, this looks like the correct value that got shifted.